app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -388,6 +389,28 @@ class ProfileEditActivityTest {
     }
   }
 
+  @Test
+  fun testProfileEdit_tooLongName_isCompletelyVisible() {
```

Comment:
Can we add a test `testProfileEdit_withLongProfileName_configChange_nameIsCompletelyVisible `

Reply:
On it

Reply:
The changes are made